src/cultionet/model.py

```diff
@@ -187,7 +189,7 @@ def fit_maskrcnn(
         devices=1 if device == 'gpu' else None,
         num_processes=0,
         accelerator=device,
-        log_every_n_steps=10,
+        log_every_n_steps=50,
```

Comment:
oh we increase this number quite a lot

Reply:
Yeah, it simply reduces the number of hits to the logger. The same metrics are tracked.